examples/wrap/main.go

```diff
@@ -107,5 +108,6 @@ func main() {
 		fmt.Fprintf(w, wrapped)
 	})
 	handler := weaver.InstrumentHandler("hello", &mux)
+	handler = otelhttp.NewHandler(handler, "http")
```

Comment:
i find it a bit weird to define `handler` and then override it with a different type of handler. 

Maybne either inline line 110 in line 111, or have different names for handlers.

Reply:
Done.